pyproject.toml

```diff
@@ -24,8 +24,9 @@ target-version = "py37"
 [tool.cibuildwheel]
 skip = ["cp36-*", "pp*", "*-musllinux_*", "*-manylinux_i686", "*_ppc64le", "*_s390x"]
 build-verbosity = 2
-test-command = "pytest {project}/tests"
+test-command = "pytest -v {project}/tests"
 test-extras = ["test", "plots"]
+test-skip = "cp37-* cp38-macosx_x86_64 *-win32"  # temporary: -win32
```

Comment:
Should `win32` be in this list of environments to skip?

Reply:
The plan is not to have `win32` skipped, but since it fails in a different way, I skipped it temporarily to focus on other failures. Will remove it before we merge PR.